test/e2e/simple_test.go

```diff
@@ -69,7 +69,9 @@ func TestE2ESimple(t *testing.T) {
 
 	totalTxs := 0
 	for _, block := range blockchain {
-		require.Equal(t, v1.Version, block.Version.App)
+		require.Equal(t, appconsts.LatestVersion,
+			block.Version.App) // the app version in the genesis is set to the
+		// latest version, so we should see this in the blocks
```

Comment:
the indentation looks off

```suggestion
		// The app version in the genesis is set to the
		// latest version, so we should see this in the blocks.
		require.Equal(t, appconsts.LatestVersion, block.Version.App) 
```

Reply:
Fixed now [4f67e83](https://github.com/celestiaorg/celestia-app/pull/3201/commits/4f67e83005d592c11871d782cf938bebf081453b)